lib/cc/analyzer/formatters/plain_text_formatter.rb

```diff
@@ -40,7 +51,13 @@ def finished
               end
 
               print(issue["description"])
+
+              if issue["severity"] && issue["severity"] != NORMAL
```

Comment:
This setup defaults to inclusively reporting the severity if it does not fit one of our expected levels [info normal critical].

Alternative: exclude severity unless `critical` or `info`